examples/api300/thunderbird/unmanaged_device.rb

```diff
@@ -0,0 +1,45 @@
+# (C) Copyright 2016 Hewlett Packard Enterprise Development LP
```

Comment:
This file should be in api300/synergy , remove the thunderbird reference and folder.